addon/src/mocks/maybe-id-url-match.js

```diff
@@ -1,34 +0,0 @@
-/**
- This is a mixin used by Mocks that might match on id or not depending if id exists
-
- @param superclass
- @constructor
- */
-const MaybeIdUrlMatch = (superclass) =>
-  class extends superclass {
-    /**
-   Used by getUrl to => this.get('id')
-
-   For mockDelete: If the id is null, the url will not include the id, and
-   can therefore be used to match any delete for this modelName
-   */
-    get(...args) {
-      if (args[0] === 'id') {
-        return this.id;
-      }
-    }
```

Comment:
didn't need to copy this over - classes making use of MaybeIdUrlMatch were already extended from a class that provided this function exacftly